app/controllers/attendances_controller.rb

```diff
@@ -15,12 +15,21 @@ def index
   def create
     @employees_todays_attendance = current_employee.todays_attendance_of_employee
     if @employees_todays_attendance.blank?
+      setting = Setting.all.first
+      today_start_time = setting.timings[Time.now.strftime('%A').downcase + '_start_time']
+      attendance_thresh = setting.attendance_time
+      if get_time_in_seconds(Time.now) > get_time_in_seconds(today_start_time.to_time) + attendance_thresh * 60
+        if current_employee.late_count.nil?
```

Comment:
we can remove this check if late_count is set to zero when creating an employee.

Reply:
tried that gives errors that's why put this check